hw09-jdbc/src/main/java/ru/otus/lantukh/jdbc/mapper/JdbcMapperImpl.java

```diff
@@ -0,0 +1,127 @@
+package ru.otus.lantukh.jdbc.mapper;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.otus.lantukh.core.dao.UserDaoException;
+import ru.otus.lantukh.core.sessionmanager.SessionManager;
+import ru.otus.lantukh.jdbc.DbExecutorImpl;
+import ru.otus.lantukh.jdbc.dao.UserDaoJdbc;
+import ru.otus.lantukh.jdbc.sessionmanager.SessionManagerJdbc;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class JdbcMapperImpl<T> {
```

Comment:
Надо реализовать интерфейс `JdbcMapper<T>`

```suggestion
public class JdbcMapperImpl<T>  implements JdbcMapper<T> {
```